src/junit.rs

```diff
@@ -10,36 +10,42 @@ use quick_xml::reader::Reader;
 use crate::testrun::{Outcome, Testrun};
 
 // from https://gist.github.com/scott-codecov/311c174ecc7de87f7d7c50371c6ef927#file-cobertura-rs-L18-L31
-fn attributes_map(attributes: Attributes) -> HashMap<String, String> {
-    return attributes
-        .filter_map(|a| {
-            if let Ok(attr) = a {
-                let bytes = attr.value.into_owned();
-                let value = String::from_utf8(bytes).unwrap();
-                let key = String::from_utf8(attr.key.local_name().as_ref().to_vec()).unwrap();
```

Comment:
When would you consider a good usecase to expect the app to crash w/ unwrap?

Reply:
I don't think we ever want to unwrap because we're exposing this as a library to the python, I don't think we want to crash, so we can let the python handle the error the way it wants